src/CLI/README.MD

```diff
@@ -16,7 +16,7 @@ You invoke the scanner by running `AxeWindowsCLI.exe`. By default, this tool get
 If you run the tool without parameters, you'll be presented with the help screen. An example follows:
 
 ```
-AxeWindowsCLI 2.1.4
+AxeWindowsCLI 2.2.0
```

Comment:
Not sure why this would be a minor version change... we've made a bug fix in a backward compatible way, which is the literal definition of a patch release from https://semver.org/#summary